app/models/play_store_rollout.rb

```diff
@@ -128,13 +141,20 @@ def halt_release!
     end
   end
 
+  def disable_automatic_rollout!
+    update!(automatic_rollout: false)
+  end
+
   def resume_release!
     with_lock do
       return unless may_start?
 
       result = rollout(last_rollout_percentage, retry_on_review_fail: true)
       if result.ok?
         start!
+        if release_platform_run.automatic_rollout?
```

Comment:
problem: I think this why you probably added the long dig to find the config in `release_platform_run` but it's not necessary.

we can just do 

```ruby
play_store_submission.conf.automatic_rollout?
```

and then delegate can be chucked 
```ruby 
delegate :automatic_rollout?, to: :release_platform_run
```